docs/routing.md

```diff
@@ -75,3 +75,20 @@ This means that instead of `import`ing `page`, you just add a `router` argument
 (In case you were wondering, the `router` argument is passed to your default-exported function by the magic code found in [`server/bundler/`](../server/bundler/).)
 
 If you're interested in single-tree rendering your section, please proceed to the [Isomorphic Routing docs](isomorphic-routing.md).
+
+## Site specific URLs
+
+When a URL in Calypso is a view that relates to a site, the `site_slug` filter should usually be the last URL fragment. This means that when you remove it you should get all sites views for the same section. For example:
```

Comment:
I *think* this should be "all sites' views" with an apostrophe to indicate the possessive, where the views belong to the sites. Am I reading that right?

Reply:
I've always thought of it as the "all sites view". The view in which all of the sites are displayed.

Reply:
Aha, I was wondering about that. :) Thanks for the clarification!